tv_apps/smart-home/locales/home.en-US.properties

```diff
@@ -1,7 +1,7 @@
 #Filter tags
 all=All
 dash=Dashboard
-tv=Live TV
```

Comment:
Is this l10n string change required by spec?
If you really need to change this, we need to replace it with another l10n key so that l10n team know there's a new string need to be translated.